components/Municipality/Municipality.tsx

```diff
@@ -72,7 +73,7 @@ function Municipality(props: Props) {
       <PageWrapper>
         <StyledContainer>
           <HeaderSection>
-            <BackArrow route="/" />
+            <BackArrow route="/geografiskt/utslappen/karta" />
```

Comment:
Not a perfect solution, but it's much better than always ending up going back to the company list.

Ideally, we should navigate back to the correct dataset too. But let's save that for the new website.

Reply:
Agreed